backstop/assets/demo.css

```diff
@@ -34,7 +34,7 @@ h3 {
 }
 
 hr {
-  border-top: 1px solid var(--iui-color-background-5);
+  border-top: 1px solid var(--iui-color-background-border);
```

Comment:
Undo this, because it can cause a lot tests images to change which makes even harder to review.

Reply:
I don't think it's because of that. I was swiping through the images and didn't see any difference between left and right.

Maybe try undoing all image changes (i.e. get them to the state before this PR) and re-running all tests and then approving again. It's possible you'll only get a few changes then.

Reply:
Count is now lower than before, but it's still a little high.  Most changes are:
- Background 4 being replaced with `iui-color-background-border` which is equal to 5 in base themes.
- Selection state opacity is less than before.
- Swapping for foreground-body in some places.
- Tooltip / skip to content color is inverting from the theme.

Reply:
Again, cant we make changes so they apply only to high-contrast themes? I thought we were fixing issues there now...
Current images do not make sense to me, at least some of them:
- Header image shows header button descripton color change, but image captures that only in one of headers (notice iModel Beta button description color) 
![image](https://user-images.githubusercontent.com/81580355/159236067-86c2cb34-f7b7-45d9-aff5-4b5450238ecf.png)
- Select arrow somehow changes size (seems like) 
before: ![image](https://user-images.githubusercontent.com/81580355/159236285-f472fc34-560c-4aba-beaf-7cfe6ba4fe13.png)
And now its color is even inconsistent with menu items above, for example
![image](https://user-images.githubusercontent.com/81580355/159236370-8cc0245c-6347-4fa1-9b85-c21e1aee4566.png)
- Not sure we can do this drastic tooltip change 
![image](https://user-images.githubusercontent.com/81580355/159236699-07feeae6-3dfe-4a8b-a165-e031f31a4a6f.png)
- This tooltip got bigger (according to new image)
![image](https://user-images.githubusercontent.com/81580355/159237319-b41d6e79-3b36-4b60-b34f-8948d8b17c4a.png)
- Is it expected that light theme images have more changes than dark (Tile border seems to change in light, but not in dark, for example)

And there might be many more.. So again, not a very nice PR to review and figure things out here :/

Reply:
The tooltip inverting has been reverted, but it does still bring _some_ change that will effect all themes.  Only change is adding a `foreground-accessory` border to help stand out against any color background.
![Screen Shot 2022-03-21 at 2 42 31 PM](https://user-images.githubusercontent.com/849817/159342165-cd7a840e-aec7-4a9d-a2b1-ba823d89f83a.png)

Reply:
Another change I brought was removing opacity from the selected secondary line of text.  This was needed to score a AAA and will help with the contrast on the base themes as well.
![Screen Shot 2022-03-21 at 2 44 19 PM](https://user-images.githubusercontent.com/849817/159342415-0700e8d2-28c5-430a-86fb-89f07391cb2e.png)

This change was also brought to menu items to match:
![Screen Shot 2022-03-21 at 2 45 42 PM](https://user-images.githubusercontent.com/849817/159342651-bb4bb732-5b9a-4047-b50b-e37aa26b41d2.png)

Reply:
> * Select arrow somehow changes size (seems like)
>   before: ![image](https://user-images.githubusercontent.com/81580355/159236285-f472fc34-560c-4aba-beaf-7cfe6ba4fe13.png)
>   And now its color is even inconsistent with menu items above, for example
>   ![image](https://user-images.githubusercontent.com/81580355/159236370-8cc0245c-6347-4fa1-9b85-c21e1aee4566.png)

I'm not seeing this change.  That first example looks like the dropdown arrow Titas implemented.  Where did you take those screenshots from?

Reply:
![image](https://user-images.githubusercontent.com/81580355/159348501-36ddca84-dd4a-4659-a722-1f88594af36a.png)

It is from your PR, where you think I am taking these from...